app/controllers/application_controller.rb

```diff
@@ -220,7 +220,7 @@ def after_sign_in_path_for(_user)
   end
 
   def signed_in_url
-    return url_for_pending_profile_reason if user_has_pending_profile?
+    return idv_verify_by_mail_enter_code_url if current_user.gpo_verification_pending_profile?
```

Comment:
`url_for_pending_profile_reason` also redirects for profiles that have `fraud_review_pending` or `fraud_rejection` status. I think we should change the behavior of that method, not remove it here. And add feature specs that confirm that users are redirected as expected - I'm surprised nothing failed.

I was going to run the in_person feature spec to get a look at idv_in_person_ready_to_verify_url and see if the user would still need to be able to get there, but my local setup won't run feature specs suddenly :-(. Do we have a banner on the account page to allow the user to access that page?

Reply:
@soniaconnolly Does the discussion at https://github.com/18F/identity-idp/pull/9410#discussion_r1364083223 help contextualize the reason for the change? My fear with modifying `url_for_pending_profile_reason` is that method is used elsewhere like `IdvController` where we'd _want_ the fraud review and fraud rejection screens to be shown, but we don't want those to be shown when signing in at IAL1.

Reply:
Oh, I see. That makes more sense, thanks.